pages/_language/events/_event/_chapter.vue

```diff
@@ -1,20 +1,30 @@
 <template>
-  <main class="chapter">
-    <h2>{{ chapter.title }}</h2>
-    <p>{{ chapter.body }}</p>
-  </main>
+  <div class="chapter">
+    <header>
+      <event-header v-bind="event" />
+    </header>
+
+    <main>
+      <h2>{{ chapter.title }}</h2>
+      <p>{{ chapter.body }}</p>
+    </main>
+  </div>
 </template>
 
 <script>
   import fetchContent from '~/lib/fetch-content';
+  import eventHeader from '~/components/event-header';
 
   export default {
+    components: {
+      eventHeader,
+    },
     head ({ params }) {
       return {
         htmlAttrs: {
           lang: params.language,
-        }
-      }
+        },
+      };
```

Comment:
In all other components we don't use trailing commas and semicolons, maybe for consistency sake don't use them here?

Reply:
Done in 7d36754f